backend/index.ts

```diff
@@ -35,3 +40,38 @@ const port = 8000;
 app.listen(8000, () => {
     console.log(`Server started: http://localhost:${port}/`);
 }); 
+
+app.get('/login/password',
+  function(req, res, next) {
+    res.render('login');
+  });
+
+app.post('/login/password',
+  passport.authenticate('local', { failureRedirect: '/login', failureMessage: true }),
+  function(req, res) {
+    console.log(req.body)
+    res.status(200).send("we logged in")
+  });
+
+const connection = mysql.createConnection({
+    host: process.env.DB_HOST,
+    user: process.env.DB_USER,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME
+  });
```

Comment:
We should also have a Prisma way of doing a connection instead of doing it like this.